Library/Format.swift

```diff
@@ -3,6 +3,8 @@ import Foundation
 import KsApi
 import Prelude
 
+public let UTCTimeZone = TimeZone(secondsFromGMT: 0)!
```

Comment:
I think this is probably fine. If it even returns `nil` we have some real problems 😬